fixtures/app_with_spm_dependencies/App/Sources/AppKit/AppKit.swift

```diff
@@ -32,7 +32,7 @@ public enum AppKit {
         _ = YAMLEncoder()
 
         // Use GoogleSignIn
-        _ = GIDSignIn.sharedInstance.hasPreviousSignIn()
+        _ = GIDSignIn.sharedInstance.configuration
```

Comment:
I'd keep using `hasPreviousSignIn()` that requires the `-ObjC` flag as an exxample. Or why was this changed?